neural_compressor/torch/algorithms/weight_only/save_load.py

```diff
@@ -228,6 +238,23 @@ def _build_woq_model(self):
         woq_model = self.original_model
         return woq_model
 
+    def _build_hqq_model(self):
+        """Build weight-only quantization model."""
```

Comment:
please fix the docstring of `_build_hqq_model` function

Reply:
Thanks, corrected at [d6dbb7e](https://github.com/intel/neural-compressor/pull/1913/commits/d6dbb7eec147f1f4e33052fd7f2e52a72cc085a1).